src/utils/clmfates_interfaceMod.F90

```diff
@@ -2426,6 +2446,7 @@ subroutine Init2(this, bounds, NLFilename)
 
     call t_startf('fates_init2')
 
+    write(iulog,*) 'Init2: calling FireInit'
```

Comment:
I'm not sure we want this write statement here. But, with testing done, we won't take it out for now. It doesn't hurt much either, although I do suspect it should at least be in a "if ( masterproc)" statement. So we should correct this in an upcoming tag.